linux/scripts/dist.sh

```diff
@@ -89,6 +89,12 @@ dpkg-source \
   --tar-ignore=keyman/resources/devbox \
   --tar-ignore=keyman/resources/docker-images \
   --tar-ignore=keyman/resources/environment.sh \
+  --tar-ignore=keyman/resources/build/ci \
+  --tar-ignore=keyman/resources/build/history \
+  --tar-ignore=keyman/resources/build/mac \
+  --tar-ignore=keyman/resources/build/pr-build-status \
+  --tar-ignore=keyman/resources/build/version \
+  --tar-ignore=keyman/resources/build/win \
```

Comment:
@ermshiperete FYI

Reply:
nit - but this kinda feels like a separate change